bazel/external/proxy_wasm_cpp_host.BUILD

```diff
@@ -14,24 +20,57 @@ cc_library(
 
 cc_library(
     name = "lib",
-    srcs = glob(
-        [
-            "src/**/*.h",
-            "src/**/*.cc",
-        ],
-        exclude = ["src/**/wavm*"],
+    # Note that the select cannot appear in the glob.
```

Comment:
Wavm has a cmake build system, why can't we build it under rules_foreign_cc rather than this handcrafted blob thing?

Reply:
WAVM is built using `rules_foreign_cc`, this here is a `BUILD` file for Proxy-Wasm C++ host implementation, which optionally includes WAVM support.

Reply:
Is there any possibility here of not checking in this long glob file?

Reply:
Do you want all the files called out separately for each case?  It will be longer, but perhaps that is OK?

Reply:
It looks like you folks are contributors to https://github.com/proxy-wasm/proxy-wasm-cpp-host. Can you add some Bazel BUILD files there?